webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Details/index.js

```diff
@@ -41,17 +41,14 @@ const DetailsTab = ({ response, status }) => (
 );
 
 DetailsTab.propTypes = {
-  response: PropTypes.shape({
-    all_parameters: PropTypes.string,
-    global_status_label: PropTypes.string,
-    interfaces: PropTypes.string,
-    name: PropTypes.string,
-  }),
+  response: PropTypes.object,
   status: PropTypes.string,
+  hostName: PropTypes.string,
```

Comment:
This could be required as well i think?

Reply:
I tried, but then  I got - 
```
Warning: Failed prop type: The prop `hostName` is marked as required in `DetailsTab`, but its value is `undefined`.
    in DetailsTab
```

which is super odd to me, I've double-checked with a debugger and at first rendering, the hostName is not undefined.

Reply:
That's odd indeed... Is it possible the router loads the page before it resolves the match somehow?

Reply:
I suspect this is somehow related to the slot itself, each tab loads as a slot. worth a further investigation, but doesn't seem related to the audit card itself. 

Regarding prop types warnings, the page has too many react warnings, which really make it harder to find corresponding warnings while developing / reviewing, so I create a cleanup [PR](https://github.com/theforeman/foreman/pull/8689)